cms/static/js/views/settings/main.js

```diff
@@ -99,11 +100,19 @@ var DetailsView = ValidatingView.extend({
         }
         this.$('#' + this.fieldToSelectorMap['entrance_exam_minimum_score_pct']).val(this.model.get('entrance_exam_minimum_score_pct'));
 
-        if (this.model.get('self_paced')) {
-            this.$('#course-pace-self-paced').attr('checked', true);
+        var selfPacedButton = this.$('#course-pace-self-paced'),
+            instructorLedButton = this.$('#course-pace-instructor-led'),
+            paceToggleTip = this.$('#course-pace-toggle-tip');
+        (this.model.get('self_paced') ? selfPacedButton : instructorLedButton).attr('checked', true);
+        if (this.model.canTogglePace()) {
+            selfPacedButton.removeAttr('disabled');
+            instructorLedButton.removeAttr('disabled');
+            paceToggleTip.text('');
         }
         else {
-            this.$('#course-pace-instructor-led').attr('checked', true);
+            selfPacedButton.attr('disabled', true);
+            instructorLedButton.attr('disabled', true);
+            paceToggleTip.text(gettext('Course pacing cannot be changed during a course run.'));
```

Comment:
@srpearce This text needs to be updated -- the pacing should not be changed after a course has started. How about "Course pacing cannot be changed once a course has started."?